### DISCUSSION TITLE
Is it possible to train my own models and contribute them to this repo?

### DISCUSSION THREAD

**Discussion by JakobEliasWagner** [jakobkruse1/emotion-recognition#30] [Q&A] [answered]
I would like to try to get some results with a couple of different datasets. Would it be possible to add these trained models to this repo, provided I get decent results from them?

**Comment by jakobkruse1** [comment, marked as answer]
Sure, I would be happy to add your model to this repository. 
Just submit a PR with the training code and feel free to send me the model files so I can evaluate them before adding.
Also, please note that tests are required for merging the PR.
Lastly, make sure that an MIT license works out for you. 
Best, Jakob
